airflow/cli/commands/celery_command.py

```diff
@@ -114,6 +116,7 @@ def filter(self, record):
         logger.handlers[:] = [below_error_handler, from_error_handler]
 
 
+@deprecated(reason="Use command from providers package", version="2.8.2")
```

Comment:
Two questions/suggestions: 

1) I think that deprecation message is misleading - it does not tell the user how to  use the command from providers package. The thing is that it should tell "please upgrade to the provider >= version of the provider where we add it"

2) Speaking of which - did you check if the new CLI command from provider is actually overridding the one from Airflow when new provider is installed? In order to test it, you will likely have to:

* prepare airflow package from main `breeze release-management prepare-airflow-package`
* prepare celery provider package from main `breeze release-management prepare-provider-packages celery`
* install new airflow and old provider (from PyPI) - and see the commands running with deprecation suggesting to install new provider
* install new airflow and new provider - celery commands should have no deprecations

Reply:
updated deprecated msg, tested msg and new provider usage